exporters/committime/README.md

```diff
@@ -27,3 +27,24 @@ Then we get commit data from the following systems through their respective APIs
 * GitHub Enterprise (including private endpoints)
 * Bitbucket _(coming soon)_
 * Gitlab _(coming soon)_
+
+## Binary Build Support
+
+OpenShift binary builds are a popular mechanism for building container images on OenShift but do not contain source code information.
+
+To support these types of build, you may annotate the build phase with the following annotations for pelorus to use:
+
+| Annotation                      | Example                           | Description                   |
+|---------------------------------|:----------------------------------|------------------------------:|
+| buildSpecRevisionGitCommit      | cae392a                           | Short or Long Git Commit Hash |
+| buildSpecSourceGitUri           | https://github.com/org/myapp.git  | URL of the Source Repository  |
+| buildSpecRevisionGitAuthorName  | joe.bloggs                        | Name of the committer         |
+
+Example command to put in your build pipeline each time you start an OpenShift `Build`:
+
+```bash
+oc annotate bc/${BUILD_CONFIG_NAME} --overwrite \
+  buildSpecSourceGitUri=${GIT_URL} \
```

Comment:
@eformat do you think it's worth matching the annotations with similar labels that are result of build process? 
Those labels are described [here](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/builds/managing-build-output#builds-output-image-labels_managing-build-output)

Here is example:

```
"io.openshift.build.source-location"
"io.openshift.build.commit.id" 
"io.openshift.build.commit.author"
"io.openshift.build.commit.ref"
```